app/plugins/adp/src/components/DeliveryProgramme/DeliveryProgrammeFormFields.ts

```diff
@@ -5,10 +5,6 @@ export const DeliveryProgrammeFormFields = [
     helperText: 'This must be unique',
     validations: {
       required: true,
-      pattern: {
-        value: /^[a-zA-Z0-9]+(?:[. ][a-zA-Z0-9]+)*$/,
```

Comment:
@kbabigumira  any top level validations.. like max length 100 characters etc?

Reply:
I can add this in with my PR as im editing these fields @DaRoszko

Reply:
We can have a limit in characters, the 'name' field on the backend has a 64 character limit when it transforms the 'title' into the format required by the templates. It might be a good idea to also keep in the limitation of adding a '.' or consecutive spaces? @danny-may @DaRoszko @kbabigumira

Reply:
I feel like this PR should be fine to go as is. The scope for this pr was to remove the error we are getting at the moment. My ticket is to make corrections to the form, and I think missing additional validation comes under that rather than the ticket for this PR

Reply:
cool